scripts/garage.coffee

```diff
@@ -18,51 +17,31 @@ module.exports = (robot) ->
 
 
 	robot.respond /(how full is garage)\s([a-zA-Z]+)\??/i, (msg) ->
-		r = request 'https://secure.parking.ucf.edu/GarageCount/iframe.aspx/', (error, response, body) ->	
-			$ = cheerio.load(body);
-			garages = [];
-			$('.dxgvDataRow_DevEx').each (i, obj) ->
-				thisGarage = {};
-				html = $(obj).html().replace(RegExp(' ', 'g'), '').split '\n'
-				for line in html
-					if line.startsWith("percent:")
-						percent = parseInt(line.replace("percent:", ''))
-						thisGarage.perc = percent
-				thisGarage.garage = ($(obj).find('.dxgv').html()).replace("Garage ", '')
-				garages[i] = thisGarage			
+		getGarages (garages) ->
```

Comment:
Maybe this is me not understanding CoffeeScript, but this seems like when a user calls "fred how full is garage x" the function `getGarages` is defined, and that's it.